### ISSUE TITLE
Rule hoistPatternLet moving let to previous lines

### ISSUE THREAD

**Issue by atfelix** [nicklockwood/SwiftFormat#370]
With `.swiftformat` and command line
```bash
$ swiftformat --version
swiftformat, version 0.39.0
$ cat .swiftformat
--rules hoistPatternLet
$ swiftformat .
```
SwiftFormat converts (notice there is no space after `case`)
```swift
func hoistPatternLet(s: String?) -> String {
    switch s {
    case.none: return "nil"
    case.some(let x): return x
    }
}
```
to
```swift
func hoistPatternLet(s: String?) -> String {
    switch s let {
    case.none: return "nil"
    case.some(x): return x
    }
}
```
With
```bash
$ cat .swiftformat
--rules spaceAroundOperators
$ swiftformat .
$ vi .swiftformat
$ cat .swiftformat
--rules hoistPatternLet
$ swiftformat .
```
it converts the same file to
```swift
func hoistPatternLet(s: String?) -> String {
    switch s {
    case .none: return "nil"
    case let .some(x): return x
    }
}
```

**Comment by nicklockwood** [comment]
@atfelix thanks for reporting. As you guessed, the lack of space is the issue. I'll push a fix asap.

**Comment by nicklockwood** [comment]
@atfelix fixed in 0.39.3

**Comment by atfelix** [comment]
@nicklockwood:  Thank you
